src/Database/LibSQLConnection.php

```diff
@@ -88,16 +69,6 @@ public function useDefaultSchemaGrammar()
         }
     }
 
-    public function createReadPdo(array $config): ?LibSQLDatabase
```

Comment:
Again, createReadPdo() is used only in ConnectionFactory, which we are not using, and it's ConnectionFactory method, not connection.